packages/design-system-react/src/components/ButtonBase/ButtonBase.tsx

```diff
@@ -151,6 +180,17 @@ export const ButtonBase = forwardRef<HTMLButtonElement, ButtonBaseProps>(
         ref={ref}
         className={mergedClassName}
         disabled={asChild ? undefined : (isDisabled ?? isLoading)}
+        aria-disabled={isDisabled ? 'true' : undefined}
+        aria-busy={isLoading ? 'true' : undefined}
+        aria-label={ariaLabel}
+        aria-labelledby={ariaLabelledby}
+        aria-describedby={ariaDescribedby}
+        aria-pressed={ariaPressed}
+        aria-expanded={ariaExpanded}
+        aria-controls={ariaControls}
+        aria-haspopup={ariaHaspopup}
+        role={asChild ? undefined : 'button'}
+        tabIndex={getTabIndex()}
```

Comment:
For native <button> elements, it's not necessary to set tabIndex={0} because they are already focusable and included in the tab order by default. Setting tabIndex={0} explicitly is redundant unless you have a specific reason, such as restoring focusability after it was removed. Typically, we only need to set tabIndex for custom interactive elements that don’t have native keyboard support. I tested with and without and the behavior is the same.


https://github.com/user-attachments/assets/c360c91e-145d-40ee-aaf7-76bf8017cd92

```suggestion
```